### DISCUSSION TITLE
Full screen / F11 support

### DISCUSSION THREAD

**Discussion by magicrobots** [aristocratos/btop4win#12] [General]
Hello! Not sure this is the place for such a question, but I'm wondering if full screen support is being considered.  Currently in Win 11, F11 is a noop.  Apologies if this is not the correct forum for a feature question!

**Comment by aristocratos** [comment]
@magicrobots 
This program runs in a terminal, I don't control if the terminal you are using supports fullscreen.
But assuming you are running in the default Windows terminal, `Alt + Enter`.

**Reply by ahillikerMDSOL** [reply]
Haha thanks, that works!
I thought the application ran as a standalone, I didn't realize it utilized the terminal.  Also I didn't know that alt + enter was fullscreen, I've always used f11!!  Thank you for teaching me multiple things today :).
